applications/PfemFluidDynamicsApplication/custom_processes/generate_new_nodes_before_meshing_process.hpp

```diff
@@ -895,11 +895,17 @@ class GenerateNewNodesBeforeMeshingProcess
     double bulkModulus=SlaveNode->FastGetSolutionStepValue(BULK_MODULUS);
     double density=SlaveNode->FastGetSolutionStepValue(DENSITY);
     double viscosity=SlaveNode->FastGetSolutionStepValue(VISCOSITY);
+    double yieldShear=SlaveNode->FastGetSolutionStepValue(YIELD_SHEAR);
+    double flowIndex=SlaveNode->FastGetSolutionStepValue(FLOW_INDEX);
+    double adaptiveExponent=SlaveNode->FastGetSolutionStepValue(ADAPTIVE_EXPONENT);
 
 
     MasterNode->FastGetSolutionStepValue(BULK_MODULUS)=bulkModulus;
     MasterNode->FastGetSolutionStepValue(DENSITY)=density;
     MasterNode->FastGetSolutionStepValue(VISCOSITY)=viscosity;
+    MasterNode->FastGetSolutionStepValue(YIELD_SHEAR)=yieldShear;
+    MasterNode->FastGetSolutionStepValue(FLOW_INDEX)=flowIndex;
+    MasterNode->FastGetSolutionStepValue(ADAPTIVE_EXPONENT)=adaptiveExponent;
 
```

Comment:
Too bad you did not use the variables @jcotela is using in the Fluid Dynamics application... Even the methods computing the apparent viscosity would have been useful.

Reply:
I am not importing the Fluid Dynamics application. I think it is not worth to do that now just for this small overlap.